bukkit/src/main/java/com/github/games647/fastlogin/bukkit/listener/protocollib/LoginSkinApplyListener.java

```diff
@@ -20,7 +20,7 @@ public LoginSkinApplyListener(FastLoginBukkit plugin) {
     }
 
     @EventHandler(priority = EventPriority.LOW)
-    public void onPlayerLogin(PlayerLoginEvent loginEvent) {
+    public void onPlayerLogin(PlayerJoinEvent loginEvent) {
```

Comment:
http://pastebin.com/jYbgFW17
this is my server log 
fastlogin is use your newest source code to build

not player is null
player.getAddress() is null (i split code to find out it)